packages/react/src/template.tsx

```diff
@@ -269,7 +263,8 @@ export function BaseTemplate({
       )
     }
     if (isStateDataState(propValue)) {
-      return render(localState[propValue.key] || propValue.initialValue, path)
+      // tbd: how to make this work?
+      return render(localState.getStream(propValue).get(), path)
```

Comment:
This worked previously, because all components re-run on `localState` change. Now, only ones that access state with `useStream` will re-render. 

As a result, props don't refresh on state change. 

There's few ways this can be done, and I think one of them are:
- pass in special object (instead of `localState`) that returns value from `localState`, but also subscribes to its change
- change will trigger component re-render

Reply:
Implemented as per above + Discord messages: https://github.com/final-ui/final/pull/76/commits/5e125232810763819d748ef88f8d43b2f6489ae4